### ISSUE TITLE
[v4.6.0] bug: Overlay .create() changed from optional options to mandatory breaking build

### ISSUE THREAD

**Issue by JeramieHallyburton** [ionic-team/ionic#18631]
# Bug Report

**Ionic version:**
[x] **4.x**

**Current behavior:**
Any OverlayController instance created using default .create() with no options passed now will not compile.

**Expected behavior:**
Can create OverlayController instances with no options passed in

**Related code:**

```
const loading = await this.loadingCtrl.create() //does not work now
```

**Other information:**
![image](https://user-images.githubusercontent.com/26013059/60204907-e978ea80-981d-11e9-981d-e8a9ec74f762.png)

Change introduced in this commit: 

https://github.com/ionic-team/ionic/commit/b40f7d36d5802251a0b44a7dd2599f19dbe977d5#diff-75f0eda4e2b7a5b00b444247a88a3232

![image](https://user-images.githubusercontent.com/26013059/60205119-6310d880-981e-11e9-88eb-53f83a21f9bc.png)

**Comment by liamdebeasi** [comment]
Thanks for the issue! 

We are currently investigating this change. While we do that, would be able to provide the output of running `ionic info` from your project's directory?

Thanks!

**Comment by JeramieHallyburton** [comment]
Ionic:

   Ionic CLI                     : 5.1.0 (C:\Users\jeram\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.6.0
   @angular-devkit/build-angular : 0.800.4
   @angular-devkit/schematics    : 8.0.4
   @angular/cli                  : 8.0.4
   @ionic/angular-toolkit        : 1.5.1

Cordova:

   Cordova CLI       : 8.0.0
   Cordova Platforms : none
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, (and 4 other plugins)

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   Android SDK Tools : 26.1.1 (C:\Program Files (x86)\Android\android-sdk)
   NodeJS            : v10.13.0 (C:\Program Files\nodejs\node.exe)
   npm               : 6.4.1
   OS                : Windows 10

**Comment by brandyscarney** [comment]
Thanks for the issue! I am going to close this as it has been fixed in Ionic, it will be in the 4.6.1 hotfix release. Please follow #18655 for updates on this. 🙂
